etna/feedback/templates/feedback/includes/prompt.html

```diff
@@ -0,0 +1,71 @@
+{% load static i18n wagtailcore_tags %}
+<div class="feedback">
+    <div class="container py-3 feedback__container">
+        <div class="row justify-content-md-center text-center feedback__content">
+            <div class="col-sm-12">
+                <form
+                    class="feedback__form"
+                    id="feedback-form"
+                    action="{% url 'feedback:submit' prompt.public_id prompt.latest_revision_id %}"
```

Comment:
Question: 
What is the purpose of  `prompt.public_id`? 
Why do we need additional an param `prompt.latest_revision_id`?

Reply:
> What is the purpose of `prompt.public_id`?

Because autoid values aren't intended to be made public - they give too much away. UUID support is such in Django that it's a common pattern to use them in public-facing functionality.

> Why do we need additional an param `prompt.latest_revision_id`?

Because prompts can be revised. Imagine the following scenario:
- User starts filling out a prompt at the bottom of the page
- Meanwhile, an admin decides to completely replace the response options for the prompt
- The user is finally happy with their comment and hits the submit button

Unless we know what version of the prompt the user was completing, how do we validate their submission? The `uuid` of the selected option would likely not be recognised and the submission would be incorrectly ignored as spam.